site/content/en/docs/reference/kueue.v1beta1.md

```diff
@@ -2525,6 +2525,49 @@ domain indicated by the values field.</p>
 
 
 
+## `WaitForPodsReadyRequeue`     {#kueue-x-k8s-io-v1beta1-WaitForPodsReadyRequeue}
```

Comment:
I'm not sure if our metrics are autogenerated but you are missing a site update for metrics.

Reply:
Unfortunately, no. Thanks!